src/components/index-page/index.tsx

```diff
@@ -16,13 +14,6 @@ import { UseCases } from "./use-cases"
 export function IndexPage() {
   return (
     <div className="gql-all-anchors-focusable bg-neu-0">
-      <Head>
```

Comment:
oh nice, I see you removed it already, lemme write a suggestion to add it to `metadata` export then